packages/frontend/core/src/components/page-list/list.css.ts

```diff
@@ -73,6 +73,7 @@ export const editTagWrapper = style({
   width: '100%',
   height: '60px',
   display: 'none',
+  zIndex: 1,
```

Comment:
useless?

Reply:
Before modification, these two buttons floated above it.
![CleanShot 2024-09-09 at 15 05 52@2x](https://github.com/user-attachments/assets/ab3bcf18-8882-4fd0-ac10-fa92b428b3cd)